plugin/README.md

```diff
@@ -181,29 +179,28 @@ I recommend to use [Apollo GraphQL](https://marketplace.visualstudio.com/items?i
     };
     ```
 
-3. Create `apollo.config.js` file in project root.
+3. Create `graphql.config.js` file in project root or supported [graphql-configs](https://graphql-config.com/usage#config-search-places).
 
     ```js
-    // apollo.config.js
+    // graphql.config.js
 
     module.exports = {
-      client: {
-        name: 'your-project-name',
-        tagName: 'graphql',
-        includes: [
-          './src/**/*.{ts,tsx}',
-          './src/__generated__/gatsby-plugin-documents.graphql',
-        ],
-        service: {
-          name: 'GatsbyJS',
-          localSchemaFile: './src/__generated__/gatsby-schema.graphql',
-        }
+      schema: ["src/__generated__/gatsby-introspection.json"],
+      documents: ["src/__generated__/gatsby-plugin-documents.graphql"],
+      extensions: {
+        endpoints: {
+          default: {
+            url: "http://localhost:8000/___graphql",
+            headers: { "user-agent": "JS GraphQL" },
+            introspect: false,
+          },
+        },
```

Comment:
We are guiding users to maintain a static configuration without exposing any http endpoint.

Is this part necessary? If it works without using it, it would better to exclude I think.

Reply:
It works without it for autocompletion and validation.
But if you want to query inside the IDE you need the endpoint otherwise you lose that functionality (see the gif)

Reply:
Oh OK it's nice